infra/base-images/base-builder/indexer/index_build.py

```diff
@@ -630,5 +658,20 @@ def main():
     shutil.move(str(snapshot), OUT)
 
 
+def get_covered_files(target: str) -> Sequence[str]:
```

Comment:
should we have a fallback if this fails?

Reply:
I don't know about a fallback. But I thinkn we shoudln't bail i there any exceptions in this.

Reply:
this is dependent on coverage builds existing right? which can break separately from regular fuzzing builds?

Reply:
Right, I'm going to add something to not quit if this fails. Is that what you mean?

Reply:
yep!